### DISCUSSION TITLE
Remove alt text for images

### DISCUSSION THREAD

**Discussion by KulkiratSinghSonrai** [remarkjs/remark#702] [Q&A] [answered]
Hello, Is there a way to disable alt text for images ?

**Comment by ChristianMurphy** [comment, marked as answer]
A plugin can be created to add or remove content as wanted, the guide on the unified site introduces how to make plugins https://unifiedjs.com/learn

More generally, this sounds like an [XY Problem](https://xyproblem.info/), why do you want to remove alt text? Alt text is important for accessibility.
